src/assets/custom/css/_sass/_service-line-case-studies.scss

```diff
@@ -0,0 +1,109 @@
+.service-line-case-studies__intro {
+  .section-intro__title,
+  .section-intro__description {
+    color: white;
+  }
+}
+
+.service-line-case-studies__cards-wrapper {
+  @include max-width--out-there;
+  align-items: stretch;
+  display: flex;
+  margin: 0 auto;
+  padding-top: 50px;
+
+  @include small{
+    flex-direction: column;
+    justify-content: center;
+  }
+  @include medium-and-large {
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+}
+
+.service-line-case-studies__card {
+  box-shadow: 0 1px 3px rgba(255, 255, 255, 0.12), 0 1px 10px rgba(0, 0, 0, 0.12);
+  display: flex;
+  flex-direction: column;
+  min-width: 230px;
+  transition: all 0.3s cubic-bezier(0.25, 0.8, 0.25, 1);
+
+  &:hover {
+    box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
+  }
+
+  @include small {
+    margin-bottom: 40px;
+    margin-left: $lateral-space--small;
+    margin-right: $lateral-space--small;
+    text-align: center;
+
+    &:nth-last-child(1) {
+      margin-bottom: 0;
+    }
+  }
+  @include medium-and-large {
+    margin-bottom: 40px;
+    max-width: 300px;
+  }
+  @include medium-large-and-extra-large {
+    margin-left: $lateral-space--small;
+    margin-right: $lateral-space--small;
+  }
```

Comment:
Looks like the margins can be on the generic. Maybe the max-width as well?

Reply:
That's a good spot with the Margins. I'll run the max width on small by Angus